webcompat/static/font/webcompat/webcompat.svg

```diff
@@ -3,7 +3,7 @@
 <svg xmlns="http://www.w3.org/2000/svg">
 <metadata>Generated by IcoMoon</metadata>
 <defs>
-<font id="webcompat" horiz-adv-x="1024">
+<font id="icomoon" horiz-adv-x="1024">
```

Comment:
Everything else looks good. Any reason for this `webcompat` -> `iconmoon` change? Maybe @magsout knows.